src/viewer/reference/native_plugins.md

```diff
@@ -393,3 +393,31 @@ Parameters are saved in local storage.
 
 A button plugin that allows to measure distances, angles and surfaces in the 2D viewer
 Measurements are saved in local storage.
+
+## Zone Editor Button
+
+- name: `zone-editor-button`
+
+This plugin allows to interact with the BIMData Zone Editor by opening the [Zone Editor plugin](#zone-editor).
+
+::: warning Note
+In order to use Zone Editor features you have to enable the [**Zone Editor plugin**](#zone-editor) (enabled by default).<br/>
+The Zone-Editor plugin will be useless if Zone Editor Button is not enabled.
+:::
```

Comment:
```suggestion
```

Not really helpfull and not 100% correct because it is possible to open the zone-editor from javascript.